python_scripts/mode_input_cli.py

```diff
@@ -22,17 +22,30 @@
 import lcm
 from lcm_types.ModeInput import ModeInput
 import warnings
+import argparse
 
 # Ignore deprecation warnings from ``lcm`` dependencies
 warnings.filterwarnings("ignore", category=DeprecationWarning)
 
 # Constants
 OFF = 0  # OFF behavior integer
 KILL = -1  # KILL behavior integer
-LCM_CHANNEL = "mode_input"  # must match channel name in control loop options
+DEFAULT_LCM_CHANNEL = "mode_input"
 
 
-def publish_mode(mode, print_msg="", lcm_ch=LCM_CHANNEL):
+def parse_channel():
+    """
+    """
+    # Parse input arguments
+    parser = argparse.ArgumentParser(
+        description='publish mode input messages via LCM.')
+    parser.add_argument('channel', metavar='channel', type=str, nargs='+',
+                        help='LCM channel name to publish on')
+    args = parser.parse_args()
+    return args.channel[0]
```

Comment:
To allow people to use this to its fullest I would likely set this up to return the args and then be used as args in the main()
This would allow us or others to add arguments more easily, for example multi channel, or rate, or something like that.

Reply:
A little more work than mode_input needs but easier to make into whatever you want

Reply:
Fair enough, fixed in 898235f93615e204682187e0ba2711b6676ef07e